File-Formats/Presentation/FAQ.md

```diff
@@ -46,3 +46,56 @@ documentation: UG
    
    You can improve the quality of converted images by specifying the image resolution. Refer – [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images)
 
+10. ## What are the NuGet packages to be installed to perform PowerPoint Presentation to PDF conversion in Linux OS?
+
+In Linux OS, you can perform PowerPoint Presentation to PDF conversion using .NET Core (Targeting .netcoreapp) application. You can refer [PowerPoint Presentation to PDF conversion NuGet packages](https://help.syncfusion.com/file-formats/presentation/nuget-packages-required#converting-powerpoint-presentation-into-pdf) to know about the packages required to deploy .NET Core (Targeting .netcoreapp) applications with PowerPoint Presentation to PDF conversion capabilities.
+
+In addition to the previous NuGet packages, the following NuGet packages need to be installed in your application.
+
+<table>
+<thead>
+<tr>
+<th width="20%">
+Version
+</th>
+<th width="40%">
+NuGet packages to install
+</th>
+</tr>
+</thead>
+<tr>
+<td>
+From v20.1.0.x 
+</td>
+<td>
+{{'[SkiaSharp.NativeAssets.Linux v2.88.0-preview.209](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux/2.88.0-preview.209)'| markdownify }}<br/>
+{{'[HarfBuzzSharp.NativeAssets.Linux v2.8.2-preview.209](https://www.nuget.org/packages/HarfBuzzSharp.NativeAssets.Linux/2.8.2-preview.209)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+From v19.4.0.x
+</td>
+<td>
+{{'[SkiaSharp.NativeAssets.Linux v2.80.2 NuGet](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux/2.80.2)'| markdownify }}<br/>
+{{'[HarfBuzzSharp.NativeAssets.Linux v2.6.1.7 NuGet](https://www.nuget.org/packages/HarfBuzzSharp.NativeAssets.Linux/2.6.1.7)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+From v18.4.0.x to 19.4.0.x
+</td>
+<td>
+{{'[SkiaSharp.NativeAssets.Linux v2.80.2 NuGet](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux/2.80.2)'| markdownify }}<br/>
+</td>
+</tr>
+<tr>
+<td>
+Before v18.4.0.x
+</td>
+<td>
+Install SkiaSharp.Linux NuGet package for .Net Core application in Linux OS. you can find the SkiaSharp.Linux NuGet package created by us from {{'[here](https://www.syncfusion.com/downloads/support/directtrac/general/ze/SkiaSharp.Linux.1.59.3-2103435070)'| markdownify }}.<br/>
```

Comment:
Install the SkiaSharp.Linux NuGet package for the .NET Core application in Linux OS. Find the SkiaSharp.Linux NuGet package created by us from {{'[here](https://www.syncfusion.com/downloads/support/directtrac/general/ze/SkiaSharp.Linux.1.59.3-2103435070)'| markdownify }}.<br/>